### ISSUE TITLE
Why are enterpictureinpicture/exitpictureinpicture being added?

### ISSUE THREAD

**Issue by domenic** [steimelchrome/document-pip-explainer#5]
It's not really clear why these events are being added in these proposal. It seems like they would be independently useful for any type of picture in picture. (And I assume they will file for any picture in picture.)

Maybe they are best done as a separate proposal?

**Comment by steimelchrome** [comment]
The HTMLVideoElement has `enterpictureinpicture` and `leavepictureinpicture` events already, but since the new PiP API doesn't happen on a particular element, we need a different way of informing the page of PiP state changes. Though thinking about it now, I should probably change `exitpictureinpicture` to `leavepictureinpicture` for consistency

**Comment by domenic** [comment]
Right, OK, so I think this ties into my question in https://github.com/steimelchrome/document-pip-explainer/issues/4#issuecomment-1017816708 about whether the design of firing on the parent Document makes the most sense, or whether firing it on the PictureInPictureWindow itself makes more sense. The latter naively seems more parallel to the HTMLVideoElement case...

+1 on aligning the names!
